### PR TITLE
[MOBL-844] opt in utils

### PR DESCRIPTION
**Pull request by rahulrvp**

### CODE REVIEW

File: android-sdk/src/main/java/com/blueshift/model/UserInfo.java [hunk 1]
```diff
@@ -81,6 +81,28 @@ private static UserInfo load(Context context) {
         return userInfo;
     }
 
+    public HashMap<String, Object> toHashMap() {
+        HashMap<String, Object> map = new HashMap<>();
+        map.put("email", email);
+        map.put("retailer_customer_id", retailer_customer_id);
```
Comment (ketanshikharebsft) [inline android-sdk/src/main/java/com/blueshift/model/UserInfo.java:87]: Please confirm if we need to send "retailer_customer_id" or "customer_id" ?
iOS is using key "customer_id".

**Review by ketanshikharebsft — APPROVED** [review]
LGTM

File: android-sdk/src/main/java/com/blueshift/model/UserInfo.java (hunk 1 above)
Comment (rahulrvp) [reply android-sdk/src/main/java/com/blueshift/model/UserInfo.java:87]: Good catch @ketanshikharebsft I will update this with the code I'm using for events.

Comment (ketanshikharebsft) [reply android-sdk/src/main/java/com/blueshift/model/UserInfo.java]: Great.. I was thinking of asking you to use the constants, looks like you replaced them with constants 👍🏻

**Pull request merged**